client/src/components/app/App.js

```diff
@@ -1,13 +1,12 @@
 import React from 'react';
+
 import style from './App.module.css';
 
 function App() {
```

Comment:
Please use the ES6 syntax here. For example- this function can be changed to 
```suggestion
const App = <div className={style.App}>...</div>;
```

Reply:
Updated